api/src/opentrons/protocol_engine/state/modules.py

```diff
@@ -1187,6 +1257,9 @@ def ensure_and_convert_module_fixture_location(
             ]
         elif model == ModuleModel.THERMOCYCLER_MODULE_V2:
             return "thermocyclerModuleV2"
+        elif model == ModuleModel.ABSORBANCE_READER_V1:
+            valid_slots = ["A3", "B3", "C3", "D3"]
+            return "absorbanceReaderV1D3"
```

Comment:
```suggestion
           addressable_areas = [
             "absorbanceReaderV1A3",
             "absorbanceReaderV1B3",
             "absorbanceReaderV1C3",
             "absorbanceReaderV1D3"
           ]
```

As written, this will allow loading the plate reader into column 3, but the geometry system would always  look for it in D3 when accessing with the gantry